Mathlib/CategoryTheory/Functor/FullyFaithful.lean

```diff
@@ -0,0 +1,408 @@
+/-
+Copyright (c) 2018 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
```

Comment:
```suggestion

! This file was ported from Lean 3 source module category_theory.functor.fully_faithful
! leanprover-community/mathlib commit 70d50ecfd4900dd6d328da39ab7ebd516abe4025
! Please do not edit these lines, except to modify the commit id
! if you have ported upstream changes.
-/
```

Reply:
Since very recently, mathport adds these headers automatically. We'll have to add them manually to older PRs.